docs/topics/settings.rst

```diff
@@ -604,6 +604,21 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
```

Comment:
Yes, a ``_DataLoss`` reason in twisted means we did not get the amount of data we expected. And that's up to the transfer decoder in twisted.

The [chunked decoder](https://github.com/twisted/twisted/blob/e8e8e8a58634e058a9f7887c9e4621e70438be8e/src/twisted/web/http.py#L1725) raises a ``_DataLoss`` exception when there is no terminating chunk.

The [identity decoder](https://github.com/twisted/twisted/blob/e8e8e8a58634e058a9f7887c9e4621e70438be8e/src/twisted/web/http.py#L1591) (non-chunked) raises a ``PotentialDataLoss`` error when there is no ``content-length`` declared and a ``_DataLoss`` error when the received content length does not match the declared ``content-length``.